queries/hoon/highlights.scm

```diff
@@ -0,0 +1,32 @@
+(number) @number
+
+(string) @string
+
+[
+  "("
+  ")"
+  "["
+  "]"
+]  @punctuation.bracket
+
+[
+  (coreTerminator)
+  (seriesTerminator)
+] @punctuation.delimiter
+
+
+(rune) @keyword
+
+(term) @constant
+
+(aura) @constant.builtin
+
+(Gap) @comment
+
+(boolean) @constant.builtin
+
+(date) @constant.builtin
+(mold) @constant.builtin
+(specialIndex) @constant.builtin
```

Comment:
Maybe `@number.builtin`? Need more context on this one

Reply:
The specialIndex rule represents 3 different ways to index an array:
- `&n` nth element
- `|n` tail of list after nth element
- `+n` slot n (tree-addressing)

all of which are converted to an numerical index, so I think `@number.builtin` would be suitable.
https://developers.urbit.org/reference/hoon/irregular#list-addressing